src/+otp/+pendulumdae/fdifferential.m

```diff
@@ -0,0 +1,19 @@
+function dstate = fdifferential(~, state, g, m, ~, ~)
```

Comment:
The convention for functions is camel case now: https://github.com/ComputationalScienceLaboratory/ODE-Test-Problems/blob/master/docs/CONTRIBUTING.md#functions